mangum/protocols/websockets.py

```diff
@@ -192,6 +189,12 @@ async def send(self, message: Message) -> None:
             await self.websocket.post_to_connection(self.connection_id, body=body)
             await self.app_queue.put({"type": "websocket.disconnect", "code": 1000})
 
+            self.logger.info(
```

Comment:
Please delete the log call here and related test. I'm going to be moving WebSockets into a separate lib and want to avoid any new additions/changes to that implementation until then.

Reply:
I ended up just going ahead with the WebSocket removal now, so you'll need to rebase on those changes because this file no longer exists.